plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/DefaultedPipelineOptionsComponent.java

```diff
@@ -139,6 +139,8 @@ private void setWidgetsEnabled(boolean enabled) {
     values.put(DataflowPreferences.STAGING_LOCATION_PROPERTY, defaultOptions.getStagingLocation());
     // TODO: Give this a separate input
```

Comment:
todo still relevant?

Reply:
It is relevant code-wise, although it is sort of a product decision whether we really want a separate input widget for this, or we will just continue to use staging directory as we do now.